index.html

```diff
@@ -384,5 +428,425 @@ <h2>Contact me</h2>
+                    <span class="span-JS">HTML</span>
+                  </div>
+
+                  <div class="div-CSS">
+                    <ul class="icons-languages-div">
+                      <li>
+                        <a href="#">
+                          <img
+                            class="JS languages-icons"
+                            src="Ellipse 3.svg"
+                            alt=""
+                          />
+                        </a>
+                      </li>
+                    </ul>
+                    <span class="span-JS">CSS</span>
+                  </div>
+                </div>
+              </div>
+              <br />
+              <div class="Frameworks-div">
+                <h2 class="Frameworks">Frameworks</h2>
+                <div class="Arrow-frameworks">></div>
+              </div>
+              <div class="Skills-div">
+                <h2 class="Skills">Skills</h2>
+                <div class="Arrow-skills">></div>
+              </div>
+            </div>
+          </section>
+
+          <!-- //////////////////footer/////////////////// -->
+
+          
+      <footer class="Contact_form_container">
+        <div class="div-BG"></div>
+        <div class="form-container">
+          <div class="Contact_form">
+            <div class="form_heading">
+            <h2>Contact me</h2>
+          </div>
+          <p class="contact-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form
+            class="form"
+            action="https://formspree.io/f/mdovglbl"
+            method="post"
+          >
+            <ul class="footer-ul">
+              <li>
+                <input
+                  class="user_name"
+                  name="User_Name"
+                  type="text"
+                  required
+                  maxlength="30"
+                  placeholder="User Name"
+                  />
+              </li>
+              <li>
+                <input
+                  class="user_email"
+                  name="User_Email"
+                  type="email"
+                  required
+                  placeholder="User Email"
+                  />
+                </li>
+              <li>
+                <textarea
+                class="massage"
+                name="massage"
+                  required
+                  placeholder="write your massage here"
+                  maxlength="500"
+                  cols="30"
+                  rows="10"
+                ></textarea>
+              </li>
+              <li class="get-in-touch-button">
+                <button class="get_in_touch" type="submit">Get in touch</button>
+              </li>
+            </ul>
+          </div>
+          </form>
+        </div>
+      </footer>
+
+  </main>
+  
+</body>
```

Comment:
**OPTIONAL** - I see that you have two sections of HTML, one for the mobile version and another for the desktop version, this is not good practice. You are required to have just one part and use CSS (media-queries) to make them responsive for both mobile and desktop. Please I strongly advise you to fix this 🙏